### PR TITLE
HPCC-13544 Add parallel child query execute test

### PR DESCRIPTION
**Pull request by jakesmith**
Signed-off-by: Jake Smith jake.smith@lexisnexis.com

### CODE REVIEW

**Comment by jakesmith** [comment]
This is the test case you gave me, with minor modifications.
@ghalliday - please review

**Comment by jakesmith** [comment]
NB: This test can cause a Thor crash at the moment, because of the unsafe aspects of the generator code (see HPCC-13543)

**Comment by AttilaVamos** [comment]
Automated Smoketest Build: success
HPCC Start: OK

| setup | hthor | thor | roxie |
| ----- | ----- | ----- | ---- |
| setup | Pass | Pass | Pass |
| setup_fetch | Pass | Pass | Pass |
| setupdict | Pass | Pass | Pass |
| setupsearchindex | Pass | Pass | Pass |
| setupsq | Pass | Pass | Pass |
| setupwordindex | Pass | Pass | Pass |
| setupxml | Pass | Pass | Pass |
 

| test | hthor | thor | roxie |
| ----- | ----- | ----- | ---- |
| parallelchild | Pass | Fail | Pass |
 
HPCC Stop: OK

File: testing/regress/ecl/parallelchild.ecl [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems.
```
Comment (ghalliday) [inline testing/regress/ecl/parallelchild.ecl]: picky: should have updated to 2015

**Comment by ghalliday** [comment]
@jakesmith should have reassigned

**Comment by jakesmith** [comment]
I'm missing a lot of email notifications from github... got 'should have reassigned' but not previous.

File: testing/regress/ecl/parallelchild.ecl (hunk 1 above)
Comment (jakesmith) [inline testing/regress/ecl/parallelchild.ecl]: ok.. but shouldn't all our copyright banners be updated annually?
(there are loads that are < 2015)

**Comment by jakesmith** [comment]
@ghalliday - have amended copyright year.

**Pull request merged**